tests/album-create.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  let artistId;
+  beforeEach(async () => {
+    const { status, body } = await request(app)
+      .post('/artists')
+      .send({ name: 'King Gizzard & The Lizard Wizard', genre: 'rock' });
```

Comment:
When pre populating the database for tests you should create the entries directly using SQL queries (like you did in the beforeEach in the artist-read.test.js file). The way you are doing it is inserting entries using your own API. The problem with this is that if the post route for artists isn't working properly then you album test would fail. I know you have tested you artist post route but it is best to not have tests relying on other parts of you app working if at all possible. Like in cruise ships when you refactored your tests with mocks, stubs and spies so they were isolated from each other.